src/utils/dates.js

```diff
@@ -3,7 +3,7 @@ import moment from 'moment'
 export function formatDate(unixTimeStamp, noTime) {
   let date = moment(unixTimeStamp).format('YYYY.MM.DD')
   if (!noTime) {
-    date = date + ' at ' + moment(unixTimeStamp).format('hh:mm:ss')
+    date = date + ' at ' + moment(unixTimeStamp).format('HH:mm:ss')
```

Comment:
as mentioned in issue #285 
we can even skip the time probably and show only the year

but if we want to show the time
I would get rid of the "seconds" (they are adding only visual clutter to the UI)
```suggestion
    date = date + ' at ' + moment(unixTimeStamp).format('HH:mm')
```